plugins/aws/README.md

```diff
@@ -18,3 +18,7 @@ It also adds it to your RPROMPT.
 * `agp`: Gets the current value of `AWS_PROFILE`.
 
 * `aws_profiles`: Lists the available profiles in the file referenced in `AWS_CONFIG_FILE` (default: ~/.aws/config). Used to provide completion for the `asp` function.
+
+## Plugin options
+
+* Set `HIDE_AWS_PROMPT=true` in your zshrc file if you want to prevent the plugin from modifying your RPROMPT.
```

Comment:
Why have it called `HIDE_` and use negated logic, when you can use `SHOW_` instead ?

Reply:
Good point, i'll get this changed 👍

Reply:
@cristim Got that changed

Also had to move the `local rprompt` declaration back into function scope outside of the if statement as It was causing RPROMPT appending issues when switching profiles, and during profile deselection.

I ended up with all my profiles in my prompt and they couldn't clear 😆